.github/workflows/_build.yml

```diff
@@ -0,0 +1,175 @@
+name: Build Docker image
+
+on:
+  workflow_dispatch:
+    inputs:
+      push:
+        description: 'Push the image to registry?'
+        default: "false"
+        required: false
```

Comment:
Curious, why is this `default: "false"` with no `type`, yet the `workflow_call:inputs:push` directly following is `default: false`, with `type: boolean`?

Reply:
Good cache, it should be boolean. Will update.